### DISCUSSION TITLE
Alias fields Translations in tabular layout

### DISCUSSION THREAD

**Discussion by d1rOn** [directus/directus#16508] [Feature Requests]
Tabular layout has possibility to create custom alias columns. It very nice feature.

My projects are often related with managing a multilingual content. For such content I use relation of translations type.

But unfortunately we cant use alias columns to view nested fields under translations relation with habitual translations display, as for now. It is looks like a bug now, even though it's not really a bug.

I have suggestion of how we can achieve this feature. Such alias fields can use translations display of the one of its parent fields (which have translations display configured) with changed template.

E.g.:

Assume we have schema:
blog_post
— id
— translations (relation to blog_post_translations, display template is configured as `{{title}}`)

blog_post_translations
— id
— blog_post_id
— languages_code
— title (raw display)
— story (raw display)

As for now, we can create alias field Translations -> Story, but it will displayed as array of story values (array of existing translations).

I suggest to display Translations field with template changed to `{{story}}`. In this case it will be displayed as expected through translations display.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
